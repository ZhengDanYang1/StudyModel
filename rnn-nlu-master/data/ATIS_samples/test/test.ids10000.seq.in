34 1 1 2 14 1 17 3 1 2 1 1 1 1 1 1 1 1 1
6 1 28 34 35 1 1 3 1 2 38 1 1 1 1 1
6 1 28 34 35 1 17 1 3 1 2 38 1
34 1 1 1 17 1 1 1 3 1 2 38 1 6 1 28
34 1 1 1 17 3 1 2 1 1 1 1 1 28 6 1 1
