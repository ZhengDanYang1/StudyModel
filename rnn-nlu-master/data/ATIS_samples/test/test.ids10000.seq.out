2 2 2 2 2 2 2 2 4 2 3 14 2 2 2 2 2 1 1
2 9 10 2 2 2 2 2 4 2 3 14 2 1 1 1
2 9 10 2 2 2 2 2 2 4 2 3 14
2 2 2 2 2 2 5 6 2 4 2 3 14 2 9 10
2 2 2 2 2 2 4 2 3 14 14 2 9 10 2 1 1
